File: question1.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int x;
    srand(time(NULL));//this prevents the same value from repeating continuously
    x= rand()% 6 + 1; //remainder between 1 and 6
    if (x==1) {
        printf("you're dead :("); //consider that 1 is the bullet cause it doesn't matter which number is assigned as we know its in a random pocket of bullet holder
    }
    else {
        printf("phew! close call");
    }
    printf("\n");
    return 0;
}

File: question 1.py
#russian roulette
import random
def roulette():
    a=input("take a shot? yes or no? ")
    if a=="yes":
        x=random.randint(1,6)
        if x==1:
            print("dead")
        else:
            print("phew!close")
    elif a=="no":
        print("goodbye")
    else:
        print("invalid input")
roulette()